ehsm_kms_service/secret_manager_apis.js

```diff
@@ -64,27 +64,54 @@ function check_rotationInterval_format(rotationInterval) {
     return true
 }
 
-//KeyId query
-const queryKeyId = (appid, keyid) => {
-    return {
-        selector: {
-            creator: appid,
-            keyid
-        },
-        fields: ['keyBlob'],
-        limit: 1
+// Query SM_MasterKey through encryptionkeyid
+const get_SM_Masterkey = async (DB, appid, encryptionKeyId) => {
+    let ret = {
+        isFind: false,
+        sm_masterKey: undefined,
+        msg: ''
     }
-}
-
-//Defaultcmk query
-const queryDefaultCMK = (appid) => {
-    return {
-        selector: {
-            appid
-        },
-        fields: ['sm_default_cmk'],
-        limit: 1
+    if (encryptionKeyId != undefined && encryptionKeyId != '' && typeof (encryptionKeyId) == 'string') {
```

Comment:
I'd like to exchange Line74{} and Line95{} as:
if (encryptionKeyId == "" || encryptionKeyId == undefined) {
......
}
else if (typeof (encryptionKeyId) == 'string') {
.......
}
else {
......
}

Reply:
The if logic will be change